t/full-stack.t

```diff
@@ -523,5 +525,8 @@ END {
     kill_driver;
     turn_down_stack;
     session->clean;
+    # prevent double execution
+    return if $? == 0;
```

Comment:
Double execution seems like a bug here, no?

Reply:
which is fixed by this line :)

Reply:
That looks more like a workaround, what it the actual cause for this getting executed twice?

Reply:
Sorry, I don't remember anymore. This PR is getting old :( If you are motivated to invest more time in the full stack test I suggest you simulate the according failure by making the internal openQA job failing with "incomplete" which should give you the desired reproducer.

Reply:
moved to https://github.com/os-autoinst/openQA/pull/2147